WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -660,6 +659,24 @@ - (void)textView:(UITextView *)textView didTypeWord:(NSString *)word
     [self.suggestionsTableView showSuggestionsForWord:word];
 }
 
+- (void)updateUIForExpandedReply
+{
+    [[self view] bringSubviewToFront:[self replyTextView]];
+    [UIView animateWithDuration:0.5 animations:^{
+        [[self navigationController] setNavigationBarHidden:TRUE];
```

Comment:
It's usual in objective-c to use `YES` and `NO` instead of `TRUE` and `FALSE`.

Reply:
There's a `setNavigationBarHidden:animated:` method you can use instead of wrapping this in an animation block (same applies to other usages of this method).

Reply:
I recently read a post where `YES` and `NO` were explained to not be the same as `TRUE` and `FALSE` in some cases.  Because this isn't a comparison that argument isn't valid.  I can swap those out.